src/components/ModalFlows/ModalFlowBase.js

```diff
@@ -96,10 +97,7 @@ function ModalFlowBase({
         content: (
           <Stepper
             steps={transactionSteps}
-            css={`
```

Comment:
are we changing this for some reason?

Reply:
Yes, since we might render more components, the margins should be adjusted accordingly.

https://github.com/1Hive/honey-pot/pull/343/files#diff-4b485407c66360ee14514f9baad0c3f18a50297893c63447d20a044007854399R176